setup.py

```diff
@@ -42,7 +42,7 @@ def get_version():
         'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
     ],
     install_requires=[
-        'Django>=1.6',
+        'Django>=1.6,<1.8',
```

Comment:
Gotcha, yea I can remove it... but it's technically broken for anybody that's wanting to test locally which is kind of a pain.